fixes #1278

sschiessl-bcp opened the pull request:
Signed-off-by: Stefan Schiessl <stefan.schiessl@blockchainprojectsbv.com>

svk31 commented:
This would be fine if the subtext was always a string, but it isn't. In the portfolio for example it's a react object, making it look like this on small screens after your changes: 

![image](https://user-images.githubusercontent.com/6890015/37252113-fc8e84de-251b-11e8-810f-53852c10eb7a.png)

You can include a check to only render the subtext when collapsed if the subText prop is a string: 

`{this.props.subText && typeof this.props.subText === "string"`

sschiessl-bcp commented:
I added functionality now so the subText can be string or object, if object it is assumed the underlying DOM model has a proper string representation.

The Activity tab had a hidden subtext span with no visibility via css. I changed that to a non-breaking space which also aligns the account tab titles, and has a proper string representation.
